### ISSUE TITLE
KeyError occured during handling of another KeyError.

### ISSUE THREAD

**Issue by JonathanNiehenke** [JonathanNiehenke/minesweeper_bot#4]
```
Traceback (most recent call last):
  File "minesweeper_ai.py", line 93, in decipher_mineweeper_cell
    cellValue = self.color_decoder[macroColor]
KeyError: (255, 255, 255, 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "minesweeper_ai.py", line 290, in <module>
    main()
  File "minesweeper_ai.py", line 170, in main
    play_minesweeper(Options)
  File "minesweeper_ai.py", line 182, in play_minesweeper
    Minefield = MinesweeperInterface(**Options)
  File "minesweeper_ai.py", line 62, in __init__
    self.read_minefield()
  File "minesweeper_ai.py", line 86, in read_minefield
    captureImage, cellIndex)
  File "minesweeper_ai.py", line 95, in decipher_mineweeper_cell
    cellValue = self.color_decoder[Object[windowX + 7, windowY + 4]]
KeyError: Pixel(red=255, green=255, blue=255)
```

Also another window other than game was brought to focus.

**Comment by JonathanNiehenke** [comment]
The traceback essentially informs that a color (255, 255, 255) is not in the predefined colors it uses to decode the screen. The other window that was brought up to focus is to blame.

The reason that window was brought up to focus is probably because it too has the same window title and was more recently in focus. I would contribute this to operator error.

**To fix** verify a pixel within the window **or** the size of the grid is divisible by the size of the tile.

**Comment by JonathanNiehenke** [comment]
Assertions of window size were included to check for foreign in commit 1bec46863c15703a398ae6c40cd9eb7b1fdc3cf5.
